src/react-native-cool-modals/ios/UIViewController+slack.swift

```diff
@@ -265,6 +271,16 @@ class PanModalViewController: UIViewController, PanModalPresentable, UILayoutSup
       config?.removeController()
     }
     config = nil
+    
+    let isSlack = self.presentingViewController?.responds(to: NSSelectorFromString("unhackParent")) ?? false
+    
+    if (isSlack) {
+      let parentConfig: RNCMScreenView? = self.presentingViewController?.value(forKey: "config") as? RNCMScreenView
+      let isHidden: Bool = parentConfig?.value(forKey: "_hidden") as? Bool ?? false
+      if (isHidden) {
+        self.presentingViewController!.dismiss(animated: false)
```

Comment:
Since you're checking on 275, we don't need the optional unwrapping here like this `self.presentingViewController?.dismiss(animated: false)`, right?